modules/svar_master/CHANGELOG.md

```diff
@@ -0,0 +1,14 @@
+# Changelog
+
+All notable changes to the `svar_master` module will be documented in this file.
+
+The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
+and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
+
+## [1.0] - 2021-12-29
+
+This release was authored by Laura Hilton.
+
+- This module integrates Manta, GRIDSS, and the HMFtools PURPLE-LINX tools into a single pipeline. It combines the Manta and GRIDSS SV calls into a single bedpe file and annotates the breakpoints by the nearest gene/locus based on a user-provided bed file. 
+- Since this module includes the GRIDSS and Manta Snakefiles but also depends on the outputs of SLMS-3, SLMS-3 MUST be run before this module. It is also highly recommended to run GRIDSS on the majority of samples prior to launching the complete pipeline. This can be accomplished by specifying `_gridss_all` as the target when launching Snakemake.  
```

Comment:
Can this maybe be duplicated as commented instruction in config? I think not all (many) users might check the CHANGELOG to read this :smile:

Reply:
Good idea